### PR TITLE
[keycastr#146] Update readme for intuitive configuration

### PR DESCRIPTION
**Pull request by mfekadu**
* adds information about setting the input monitoring security preference as identified by issue #146 https://github.com/keycastr/keycastr/issues/146

* adds information about what the "display all keystrokes" checkbox does as discussed on Gitter https://gitter.im/keycastr/Lobby

Co-Authored-By: Elia Schito <elia@schito.me>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,29 @@ In order to work, KeyCastr must be included in the list of applications with Acc
 
 To add KeyCastr to the list click the <kbd>&plus;</kbd> button and select KeyCastr from the file system.
 
+> NOTE: If KeyCastr is already in the list, then click the <kbd>&minus;</kbd> button and add KeyCastr again to be certain that the right application is chosen.
+
+## Input Monitoring
+
+Also, KeyCastr needs your explicit permission to to monitor inputs. You can set this in the Input Monitoring section under the Security & Privacy system preferences.
```
Comment (akitchen) [inline README.md]: Hmm, I'm not sure this is actually true. I haven't needed to do this on my machine and I'm on macOS 10.15.3

Comment (akitchen) [reply README.md]: This was needed in a clean installation of KeyCastr on macOS 10.15 Catalina, however in that case it replaces the need for Accessibility permissions. So this instruction could use further refinement.

**Comment by svenjacobs** [comment]
I'm not able to add KeyCastr to Input Monitoring as +/- buttons are missing. macOS Catalina 10.15.3.

Screenshot 2020-03-06 at 13 56 00

**Comment by akitchen** [comment]
@svenjacobs I believe in this case you need to remove it from the Accessibility list (if present), run the app & see the error, and then the OS will automatically add it to the Input Monitoring list. At that point you need to authenticate & check the box next to it to enable it. HTH!

File: README.md (hunk 1 above)
Comment (mfekadu) [reply README.md]: We could change the wording to “KeyCastr _may_ need your explicit permission on certain versions of macOS”

**Comment by svenjacobs** [comment]
I removed KeyCastr from Accessibility list and restarted the application. This is what happened:

Screenshot 2020-03-09 at 12 52 24

Dialogs were overlapping so I moved them side by side. Now I was able to add KeyCastr to Input Monitoring but I guess the onboarding process could be improved 😉

**Pull request merged**